pr_check.sh

```diff
@@ -22,14 +22,13 @@ set -exv
 APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
 COMPONENT_NAME="insights-content-template-renderer"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
 IMAGE="quay.io/cloudservices/insights-content-template-renderer"
-COMPONENTS="insights-content-template-renderer"  # space-separated list of components to laod
+COMPONENTS="ccx-data-pipeline ccx-insights-results dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer"  # space-separated list of components to load
```

Comment:
do we need all those for servicelog?

Reply:
I think so, but I'll check once more. It does not seem so, but for example we need aggregator endpoints for the tests themselves and dvo-writer is needed to run the aggregator.

Reply:
And do you know why aggregator is needed? For which tests?